app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
In your function definition, you are using type hinting for the `friends` parameter and the return type of the function, which is good. However, you are not using type hinting for the `cafe` parameter. It's always good practice to use type hinting for all parameters in a function definition. This makes your code more understandable and easier to debug. Here's how you can do it:

python
def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
    ...
```